package.json

```diff
@@ -30,7 +30,8 @@
     "react-stripe-checkout": "^2.6.3",
     "react-time-ago": "^5.0.7",
     "stylus": "^0.54.7",
-    "twitter-text": "^3.0.0"
+    "twitter-text": "^3.0.0",
+    "typescript": "^3.6.4"
```

Comment:
I think TS should be a dev dependency

Reply:
@olpeh you're right!
@rikukissa should I fix it or should I remove https://github.com/koodiklinikka/koodiklinikka.fi/pull/62/commits/a13cfac8bb0949bd00345f11ae802b319fd8bea5

Would you like it to be done with rebase & force push or with a new commit?

Reply:
I think as Olavi's PR fixes it, it can be dropped from this PR. I don't mind how you make the change :) New commit is just fine as that's probably the easiest